package.json

```diff
@@ -219,7 +220,7 @@
     "zlib": "browserify-zlib",
     "console": "console-browserify",
     "constants": "constants-browserify",
-    "crypto": "react-native-crypto",
+    "crypto": "crypto-js",
```

Comment:
could you remove this from this PR it goes a bit out of the scope of the feature :+1: 
@valpinkman @MortalKastor to address the update needed and create a separate PR for it

Reply:
ok for me, do I need a Jira Ticket ? Which squad ? or only PR with @valpinkman and @MortalKastor mentionned ?